core/src/serve_repair.rs

```diff
@@ -151,7 +151,8 @@ impl RequestResponse for AncestorHashesRepairType {
 #[derive(Default)]
 struct ServeRepairStats {
     total_requests: usize,
-    dropped_requests: usize,
```

Comment:
@behzadnouri I saw you asked for this to be tracked for pager-duty. I'll follow up to have this updated. Do you know if there are other consumers which may be tracking `dropped_requests`?

Reply:
I don't see it in any of grafana dashboards, so probably not.

@c3pko is this one of the metrics in your spreadsheet?